keras_eval/utils.py

```diff
@@ -257,10 +258,19 @@ def create_image_generator(data_dir, batch_size, model_spec):
     Returns: Keras generator without shuffling samples and ground truth labels associated with generator
 
     '''
-    test_gen = image.ImageDataGenerator(preprocessing_function=model_spec.preprocess_input)
+    if data_augmentation is None:
+        test_gen = image.ImageDataGenerator(preprocessing_function=model_spec.preprocess_input)
+        generator = test_gen.flow_from_directory(data_dir, batch_size=batch_size,
+                                                 target_size=model_spec.target_size[:2],
+                                                 class_mode='categorical', shuffle=False)
+    else:
+        test_gen = AugmentedImageDataGenerator(preprocessing_function=model_spec.preprocess_input,
+                                               data_augmentation=data_augmentation)
+        generator = test_gen.flow_from_directory(data_dir, batch_size=1,
```

Comment:
I guess we hardcode `batch_size` to be 1 for the `data_augmentation` case for computation cost issues, right?

Reply:
Because it's easier to process later if every image is augmented separately. 
Batch sizes are probably the same big as they were but with augmented copies of only one image instead of having different images.